src/index.html

```diff
@@ -4,9 +4,31 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- icons -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <title>Todolist</title>
 </head>
 <body>
-    <h1>Todolist</h1>
+    <main>
+        <section class="container">
+            <div class="title">
+                <h1>today's todo</h1>
+                <i class="fa-solid fa-repeat fa-beat-fade icon"></i>
+            </div>
+            <div class="add">
+                <input type="text" name="" id="" placeholder="Add To Your List">
+                <i class="fa-solid fa-arrow-left icon"></i>
+            </div>
+            <div class="todolist">
+                <div class="list">
+                    <div class="content">
+                        <input class="box" type="checkbox" name="" id="">
+                        <input type="text" name="" id="" value="testone">
+                    </div>
+                    <i class="fa-solid fa-ellipsis-vertical icon"></i>
+                </div>
+            </div>
```

Comment:
- [ Required ] I think it would be better if you add the `Clear all completed` button in your design so that your todo-list is consistent with the project wireframe provided for this project. Please see the screenshots below and rectify this.

| Microverse design | Your design |
|-----------------|--------------------|
|![Screenshot from 2023-06-07 01-35-44](https://github.com/Othniel2471/todolist-mv/assets/16392046/ef66b5ae-8008-4015-9088-5e3d546d34af)|![Screenshot from 2023-06-07 01-40-14](https://github.com/Othniel2471/todolist-mv/assets/16392046/26b343fa-bd94-43aa-86b0-15650e3a5dcf)|